beacon-chain/sync/metrics.go

```diff
@@ -82,6 +82,13 @@ var (
 			Buckets: []float64{5, 10, 50, 100, 150, 250, 500, 1000, 2000},
 		},
 	)
+	rpcBlobsByRangeResponseLatency = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "rpc_blobs_by_range_response_latency_milliseconds",
+			Help:    "Captures total time to respond to rpc BlobsByRange requests in a milliseconds distribution",
+			Buckets: []float64{5, 10, 50, 100, 150, 250, 500, 1000, 2000},
```

Comment:
We probably want to visit this later but having them the same as block makes sense to me